lighthouse-core/test/audits/user-timing-test.js

```diff
@@ -41,28 +36,4 @@ describe('Performance: user-timings audit', () => {
       assert.equal(auditResult.details.items[1].duration, undefined);
     });
   });
-
-  it('doesn\'t throw when user_timing events have a colon', () => {
```

Comment:
it seems like this was maybe a regression test added at some point. Any reason to delete instead of switching over to checking `items`? (looks like the same `name` filter should work)